Sources/App/Controllers/Registration/RegistrationFormContrloller.swift

```diff
@@ -0,0 +1,30 @@
+import HTTP
+import Vapor
+import Fluent
+
+final class  RegistrationFormController {
```

Comment:
Please look at the file name(not class name). It's `RegistrationFormContrloller`, fix a typo)

Reply:
oops, thank you